pkg/docker/docker.go

```diff
@@ -32,6 +32,19 @@ const (
 	workingDir = "/in-toto" // Where we expect to copy in-toto artifacts to
 )
 
+// GetGUN returns the Globally Unique Name for a reference image name
+func GetGUN(name string) (string, error) {
+	r, err := reference.ParseNormalizedNamed(name)
+	if err != nil {
+		return "", err
+	}
+	repo, err := registry.ParseRepositoryInfo(r)
```

Comment:
I don't understand what this second function call does for you? It seems that you can return r.Name directly and the test still passes.